configs/_base_/backends/ipu.py

```diff
@@ -0,0 +1,3 @@
+backend_config = dict(type='ipu', precision='fp16', output_dir='/localdata/cn-customer-engineering/qiangg/mmdeploy_popef_hub',
```

Comment:
Do we have to fix `output_dir`? can we save the output to the work directory?

Reply:
ok